### DISCUSSION TITLE
Discussions

### DISCUSSION THREAD

**Discussion by MedouneSGB** [MedouneSGB/Waxtane#5] [General]
### Let's Chat ! 😁

**Comment by MedouneSGB** [comment]
Hello @dofbi
Après ajout d'un enregistrement,
l'action se mettait en marche, mais il n'y a aucun changement dans les fichiers cible.

**Reply by dofbi** [reply]
Hi @MedouneSGB 
Fixe #9 👍🏾

**Comment by MedouneSGB** [comment]
Hello guys j'ai changé l'icons du site.
Il est assez simple, mais un peu plus expressif !
![image](https://user-images.githubusercontent.com/40875400/167690583-f9912b7a-a88a-46e8-896c-8573a6fd52eb.png)

**Comment by dofbi** [comment]
Cool en mode transparent 👍🏾

**Comment by MedouneSGB** [comment]
Great idea !!!🔥
Added ✨

**Comment by MedouneSGB** [comment]
![icon](https://user-images.githubusercontent.com/40875400/167927924-a45ea5d1-3080-4dc6-99b3-5c4bcc762b89.png)
